tools/migrate_1121403_roomparticipants.js

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+ * Migrate the room participants from the old format to the new one.
+ *
+ * Previously, a "roomparticipants.{roomToken}" key was holding the list of
+ * participants, wich their value. The Redis data-type was a hash.
+ *
+ * Bug 1121403 changed that in order to have expiration events on the
+ * participants; the data type is now a set, refering to all the participants
+ * which are stored in independent keys.
```

Comment:
or something like *to all participants ids. Particpants details are now stored with their own independent key.*